src/lib/pod/src/lib.rs

```diff
@@ -325,3 +325,7 @@ unsafe impl Pod for libc::utmpx {}
 unsafe impl Pod for libc::utsname {}
 unsafe impl Pod for libc::winsize {}
 unsafe impl Pod for libc::clone_args {}
+
+// linux-raw-sys types
+unsafe impl Pod for linux_raw_sys::general::__user_cap_header_struct {}
+unsafe impl Pod for linux_raw_sys::general::__user_cap_data_struct {}
```

Comment:
Yup, agreed we should try to have everything use `linux-api` for consistency. For some types `linux-api` creates safe(r) wrappers around the raw bindings, but IMO adding and exposing the raw binding types is probably adequate for this PR.

At some point it might make sense for `linux-api` to use  `linux-raw-sys` instead of generating bindings itself, but we currently rely on the name mangling done in our binding-generation for compatibility with C code. (Even if we don't need to expose these new types to C code, I think for now we want everything to go through this path for consistency)

Hopefully the docs and examples in `linux-api` are sufficient, but LMK if you have questions!

Reply:
I saw this `linux-api` thing when I implemented this, but, since it requires some work, I used `linux-raw-sys` first and see what you guys think and yeah let's go with `linux-api`.